backend/clickhouse/traces.go

```diff
@@ -281,6 +283,41 @@ func (client *Client) ReadTraces(ctx context.Context, projectID int, params mode
 	}, nil
 }
 
+func (client *Client) ExistingTraceIds(ctx context.Context, projectID int, traceIDs []string) ([]string, error) {
```

Comment:
to check my understanding here, essentially this is a way to bulk query for traces by id, but only returning the ids (rather than the full trace rows)